extensions/blocks/gif/edit.js

```diff
@@ -143,7 +143,7 @@ class GifEdit extends Component {
 					onChange={ value => setAttributes( { searchText: value } ) }
 					value={ searchText }
 				/>
-				<Button isLarge onClick={ this.onSubmit }>
+				<Button isSecondary onClick={ this.onSubmit }>
```

Comment:
I think this is problematic on dark themes.

**Before**
![image](https://user-images.githubusercontent.com/426388/72548730-91740500-388f-11ea-8f1a-cf0fe286ee2e.png)

**After**

![image](https://user-images.githubusercontent.com/426388/72548717-8ae58d80-388f-11ea-9007-9a1991a68b27.png)

Reply:
Thanks! This should be fixed with: https://github.com/Automattic/jetpack/pull/14371#issuecomment-575345537